tools/crm_mon.c

```diff
@@ -2022,34 +2022,6 @@ crm_diff_update_element_v2(xmlNode *change, void *userdata)
     return pcmk_rc_ok;
 }
 
-static void
-crm_diff_update_v2(const char *event, xmlNode * msg)
-{
-    xmlNode *wrapper = pcmk__xe_first_child(msg, PCMK__XE_CIB_UPDATE_RESULT,
-                                            NULL, NULL);
-    xmlNode *diff = pcmk__xe_first_child(wrapper, NULL, NULL, NULL);
-
-    pcmk__xe_foreach_child(diff, NULL, crm_diff_update_element_v2, NULL);
-}
-
-static void
-crm_diff_update_v1(const char *event, xmlNode * msg)
-{
-    /* Process operation updates */
-    xmlXPathObject *xpathObj = xpath_search(msg,
-                                            "//" PCMK__XE_CIB_UPDATE_RESULT
-                                            "//" PCMK__XE_DIFF_ADDED
-                                            "//" PCMK__XE_LRM_RSC_OP);
-    int lpc = 0, max = numXpathResults(xpathObj);
-
-    for (lpc = 0; lpc < max; lpc++) {
-        xmlNode *rsc_op = getXpathResult(xpathObj, lpc);
-
-        handle_rsc_op(rsc_op, NULL);
-    }
-    freeXpathObject(xpathObj);
-}
-
```

Comment:
This could definitely apply to newer remote nodes connecting to an older cluster node, broadened T301